qforce/qm/orca.py

```diff
@@ -0,0 +1,624 @@
+import os.path
+
+from colt import Colt
+import numpy as np
+from ase.units import Hartree, mol, kJ, Bohr
+#
+from .qm_base import WriteABC, ReadABC
+from ..elements import ATOM_SYM
+
+
+class Orca(Colt):
+    _user_input = """
+
+    charge_method = esp :: str :: [cm5, esp]
+    
+    # QM method to be used for geometry optimisation
+    opt_method = XTB2 :: str
+    # For high-throughput use XTB2
```

Comment:
The comments should come before the keyword. They are then added to the documentation directly.